### ISSUE TITLE
Option to enable date columns to link to daily notes

### ISSUE THREAD

**Issue by Yohe-Am** [trey-wallis/obsidian-dashboards#589]
### Problem

I use NLT to track my personal finance and make use of a date column. Say I come across a suspicious 2k entry from 2 months ago that I don't remember spending. I usually write down the actual daily activities in the daily not entry so that's where I'd go next to solve the mystery. Right now, I can use the quick switcher and that works well enough but it can be smoother.

### Solution

An option to enable date columns to be links to the daily note for that date. Format ought be configurable I suppose. It'd be even better if this could integrate the native preview on hover function of Obsidian.

### Additional details

I was going make this part of #588 to avoid being spammy but I reckoned they're distinct enough.

**Comment by trey-wallis** [comment]
Hi @Yohe-Am. Could you give me an example of this? Does DataLoom 8.14.0 provide this functionality?

**Comment by Yohe-Am** [comment]
@trey-wallis Nah, 8.14.0 doesn't support the mentioned functionality. In fact, updating to 8.14.0 seems to have garbled dates in my already existing looms. I'm only seeing Nans now.

Not sure how to illustrate an example though. Say you have rows in your loom with date columns. Say you also have notes in your vault titled in the format `[[2023-11-25]]`. It'd be cool to be able to toggle some option in the date column setting so that any  rows with values `2021/11/25` point to that note.

**Comment by trey-wallis** [comment]
Ohh I understand now. Thanks

**Comment by trey-wallis** [comment]
As for the garbling, that shouldn't have happened.

Can you check the loom file and see what it has set to the 'dateTime' value
for each cell?

**Comment by Yohe-Am** [comment]
The `dateTime` values are all empty strings. 

In an attempt to remedy it, I undid the changes from the update from the Git history. The old version of the file opens still but I'm still seeing Nans. Then I noticed that, unlike previously, DataLoom didn't update the old file format to the new one automatically when I opened it. I discovered that adding new rows triggers this but even afterwards, the old rows have empty strings for `dateTime`. New rows have properly functioning `dateTime`s ofc.

**Comment by trey-wallis** [comment]
I am sorry that this happened. I found the issue and it is fixed now for
8.15.2. Any future looms that you update from before 8.13.0 will not have
this problem. Otherwise you will have to update them using previous data.
